### DISCUSSION TITLE
Scene will suddenly not open

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#2911] [General Questions & Discussion] [answered]
I have a scene that was working fine until my latest publish, when it suddenly wouldn't let me open it.  The screen just went black when I clicked on 'open.'  The scene is under the size limit and was fine before. What normally causes this? Can you open this on your end and what are you seeing?  (https://hubs.mozilla.com/scenes/XMCiFGw)

**Comment by eric-cornellvel** [comment]
Hi @hilarytsai, I also encountered the same issue. The scene that I designed and published from Spoke appear as a black screen.

**Reply by carlagnyu** [reply]
I am having the same issue. Today, Aug 22, is the first day this has happpened. Anything I've built in spoke, when i click on view in hubs or publish to hubs, I only get a black screen on launch. I've tested in firebox and chrome on two different computers.

**Reply by hilarytsai** [reply]
@eric-cornellvel @carlagnyu  This is very strange indeed.  I had to go to a previous Url for the scene that I'd published earlier, which did work and showed the updated changes.  But now I'm having trouble with that, too. It just stays on loading objects indefinitely.

**Comment by carlagnyu** [comment]
*Also, the desktop app is no longer working!

**Comment by emclaren** [comment, marked as answer]
Hi @hilarytsai thanks for reporting this. I've tested this scene on several browsers and it appears to be working. It is a very large scene though so it takes a little while to open. 

Can you or @carlagnyu confirm if this is still happening for you folks? I wonder if there was an issue happening in the back end causing this for a few days which is perhaps now resolved.

**Reply by carlagnyu** [reply]
oh, all working for me now! Had an event open in hubs last weekend :) Thx for checking in!
